lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -325,7 +324,7 @@ export default class BitcoinProcessor {
       throw new RequestError(ResponseStatus.BadRequest, ErrorCode.BlockchainTimeOutOfRange);
     }
 
-    const currentBlockNumber = await this.getCurrentBlockHeight();
+    const currentBlockNumber = this.lastProcessedBlock ? this.lastProcessedBlock.height : await this.getCurrentBlockHeight();
```

Comment:
Consider just assigning to `this.lastProcessedBlock`, since `initialize()` would have already been called.